app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -771,7 +769,8 @@ public void run() {
                             final View view = _recyclerView.getChildAt(i);
                             final TextView textView = view.findViewById(R.id.opoc_filesystem_item__title);
                             if (data.file.getName().equals(textView.getText().toString())) {
```

Comment:
I already have code to blink any view. I think you should just use that. If necessary, modify it change the background in this way.

Reply:
I think it may be unnecessary to use that complex animation here.